Date: Tue, 10 Jan 89 18:03:57 EST
To: NeWS-makers@brillig.umd.edu
Subject: FrameMaker 1.3 psview...
From: siegel@hc.dspo.gov  (josh Siegel)


I noticed that FrameMaker produced files that couldn't be viewed by
psview under NeWS...

If you apply the following to .../FrameMaker/.makerinit/postscript_prolog,
you can both psview and print the postscript files.

Note, the setpattern command has been gutted since the functions
that would be needed havn't been implemented under NeWS.

This has only been tested with FrameMaker 1.3..

			--Josh Siegel

*** postscript_prolog.working   Tue Jan 10 13:20:21 1989
--- postscript_prolog   Tue Jan 10 13:44:48 1989
***************
*** 245,250 ****
--- 245,266 ----
         {} settransfer
         offbits offbits onbits add div setgray
        } bind def
+
+       version (1.1) eq {
+               /defaultmatrix {
+                       gsave
+                                       initmatrix
+                                       currentmatrix
+                       grestore
+               } bind def
+
+               /setpattern {
+                       pop pop pop pop
+
+                       .25 setgray
+               } bind store
+       } if
+
  /grayness {
        setgray
        orgxfer cvx settransfer

